evergreen/install_baas.sh

```diff
@@ -97,7 +109,7 @@ esac
 # Allow path to $CURL to be overloaded by an environment variable
 CURL="${CURL:=$LAUNCHER curl}"
 
-BASE_PATH="$(cd $(dirname "$0"); pwd)"
+BASE_PATH="$(cd "$(dirname "$0")"; pwd)"
```

Comment:
Thanks for fixing the shellcheck warnings. 👍 
There is one more at line 128:
`set -- $PASSED_ARGUMENTS` -> `set -- "$PASSED_ARGUMENTS"`
And, an extra (unsupported) getopts flag at line 123:
`PASSED_ARGUMENTS=$(getopt w:s:b: "$*") || usage` -> `PASSED_ARGUMENTS=$(getopt w:b: "$*") || usage`
_There is no handler for `s` (and an argument) in the case statement_

Reply:
Adding quotes around `$PASSED_ARGUMENTS` breaks it, as expansion is actually needed. The correct way to do it is to use an array, but that's a less trivial change to make.